src/App.tsx

```diff
@@ -1,7 +1,44 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type Props = {};
+type State = {
+  text: string;
+};
+
+export class App extends React.Component<Props, State> {
+  state = {
+    text: '',
+  };
+
+  componentDidMount(): void {
+    this.handleState();
+  }
+
+  componentDidUpdate(): void {
+    document.addEventListener('keyup', this.handleKeyUp);
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keyup', this.handleKeyUp);
+  }
+
+  handleState = () => {
+    this.setState({
+      text: 'Nothing was pressed yet',
+    });
+  };
```

Comment:
[checklist] Make the App a class component with pressedKey in the state.

Also you need keep in state current key, but not a full sentence. You can use conditional rendering to display the desired text